.github/workflows/local-node-test.yaml

```
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+name: "Local Node E2E Tests"
+
+on:
+  pull_request:
```

```suggestion
  pull_request:
      types:
      - opened
      - reopened
      - synchronize
```

I'd suggest adding these standard types to the pull request triggers. This is our standard practice.